substrate/frame/mixnet/src/lib.rs

```diff
@@ -168,12 +163,9 @@ fn twox<BlockNumber: UniqueSaturatedInto<u64>>(
 // The pallet
 ////////////////////////////////////////////////////////////////////////////////
 
-#[frame_support::pallet(dev_mode)]
+#[frame::pallet(dev_mode)]
```

Comment:
This `dev_mode` is not really good for production as well 🙈  not in scope for now, but should be removed later.

Reply:
I see, will have to dig in the code to understand why but we could create an issue for it if there's isn't one already.